Source/ParserProperty.swift

```diff
@@ -24,19 +24,17 @@
 
 import Foundation
 
-/**
-    Defines a Swift object type used to extract values from a JSON document.
-
-    - string:     Represents a Swift `String` type.
-    - uint:       Represents a Swift `UInt` type.
-    - int:        Represents a Swift `Int` type.
-    - float:      Represents a Swift `Float` type.
-    - double:     Represents a Swift `Double` type.
-    - bool:       Represents a Swift `Bool` type.
-    - array:      Represents a Swift `Array` type.
-    - dictionary: Represents a Swift `Dictionary` type.
-    - url:        Represents a Swift `URL` type.
-*/
+/// Defines a Swift object type used to extract values from a JSON document.
+///
+/// - string:     Represents a Swift `String` type.
+/// - uint:       Represents a Swift `UInt` type.
+/// - int:        Represents a Swift `Int` type.
+/// - float:      Represents a Swift `Float` type.
+/// - double:     Represents a Swift `Double` type.
+/// - bool:       Represents a Swift `Bool` type.
+/// - array:      Represents a Swift `Array` type.
+/// - dictionary: Represents a Swift `Dictionary` type.
+/// - url:        Represents a Swift `URL` type.
 public enum ParserPropertyType {
```

Comment:
`SchemaPropertyProtocol`? Apple wants you to replace `Type` with `Protocol` now I believe.